src/annotate.ts

```diff
@@ -43,43 +54,48 @@ export default async function annotate({payload, _ajv = ajv, _schema = schema}:
     json: true
   };
 
-  let json: any = await _request(options)
+  const json: any = await _request(options)
     .then((res: any) => {
       return res;
     })
     .catch((err: any) => {
-      //console.error('_request error: ' + err);
       throw new Error('_request error: ' + err);
     });
 
-  json = json[0];
-  //console.log(JSON.stringify(json, undefined, 2));
-  // if there is no match, returns undefined
-  // if there are multiple matches, returns only the first one.. check this is appropriate.
+  // if there is no match, returns [undefined]
+  // if there are multiple matches, returns all.
 
-  if (json) {
+  if (json.length > 0 && json[0] !== undefined) {
+    return json.map((x: any) => {
```

Comment:
Is there a better way to check whether 
```
json !== [undefined]
```

Reply:
Why do you need to check for  the first entry in the array? `json[0] !== undefined`. I would check if it's an array with [Array.isArray](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray), you can skyp the null & undefined checks as the function will do it for you.

Reply:
The request returns `[undefined]` if the term isn't found. So checking it is an array isn't enough.